recipes/xorg-macros/all/test_package/conanfile.py

```diff
@@ -7,12 +7,14 @@ class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     exports_sources = "configure.ac", "Makefile.am",
     generators = "pkg_config"
+    test_type = "explicit"
 
     @property
     def _settings_build(self):
         return self.settings_build if hasattr(self, "settings_build") else self.settings
 
     def build_requirements(self):
+        self.build_requires(self.tested_reference_str)
```

Comment:
There is only one cross-compiling scenario now in CCI: we build apple-M1 binaries in a regular apple-x86 machine. In that scenario build-profile is x86 while host-profile is M1. For that configuration, with this recipe, we are testing here the x86 binaries (build-profile) while building M1 ones (host-profile).

This is working because we build first apple-x86 binaries and then apple-M1 binaries... and the x86 binary required here is downloaded from the remote. M1 binary is not really tested.